src/cli.ts

```diff
@@ -104,7 +104,7 @@ const arrayToEnum = (enums: string[], enumName: string) => {
   }
 
   if (name.includes(".")) {
-    name = name.replace(/\./g, "_");
+    name = parseValue(name);
```

Comment:
Now if you are handling it with using `parseValue` method, the `if` statement with includes check is IMO unnecessary. That's because if none other options are turned on, the output value will be the same as the input due to all those `if` statements within `parseValue` function and we don't want to normalize only if dot exists within the name value.

But now i have another concern, that's because this can be somehow a breaking change for already generated enums. If we turn on `normalize` and `uppercase` in the same time all generated and exported enum names will change to uppercase. I'm not sure if this will be expected outcome :thinking: So i'm wondering if it's a good way to handle enum names and enum keys in the same manner depending on passed options.

Reply:
ok, let's go back to the previous solution which will handle only the dot case. I'll release it as 0.5.1 and think about how to handle the rest of cases during normalization process. Because I see that not everyone will expect to have UPPERCASED export names by just using single option flag. So i suppose i will split that somehow between two separate flags.